discovery-infra/test_infra/helper_classes/nodes.py

```diff
@@ -93,8 +92,7 @@ def get_random_node(self):
     def shutdown_all(self):
         self.run_for_all_nodes("shutdown")
 
-    def start_all(self):
-        static_ips_config = env_variables.get('static_ips_config')
+    def start_all(self, static_ips_config):
```

Comment:
static_ips_configs should have a default

Reply:
agreed should default to False

Reply:
👍🏻